tests/repo_error/portal_objects/file_reference.yaml

```diff
@@ -12,11 +12,15 @@ secondary_files:
 status: uploading
 uuid: 1936f246-22e1-45dc-bb5c-9cfd55537fe7
 accession: GAPFIXRDPDK5
+category: Sequencing Reads
```

Comment:
You might want to make this an enum portal side so we can control these values, could get messy with time especially if we have these coming from external submitters

Reply:
if not we definitely should